inc/structured-data-blocks/class-how-to-block.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Structured_Data_Blocks
+ */
+
+/**
+ * Class WPSEO_How_To_Block
+ */
+class WPSEO_How_To_Block {
+	/**
+	 * Registers the how-to block as a server-side rendered block.
+	 *
+	 * @return void
+	 */
+	public static function register() {
```

Comment:
I prefer to use the non-static way, because all the methods depend on each other.